azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -123,6 +123,15 @@ private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessag
     }
   }
 
+  private String getJobOrFlowName(SlaOption slaOption) {
+    String jobName = (String) slaOption.getInfo().get(SlaOption.INFO_JOB_NAME);
+    if (org.apache.commons.lang.StringUtils.isNotBlank(jobName)) {
```

Comment:
consider importing org.apache.commons.lang.StringUtils ?

Reply:
Not possible because there's a class called `StringUtils` in the same package (`azkaban.utils`) as `Emailer`.